pkg/internal/apis/certmanager/validation/certificate_test.go

```diff
@@ -426,6 +427,39 @@ func TestValidateCertificate(t *testing.T) {
 				},
 			},
 		},
+		"valid certificate with only email SAN": {
+			cfg: &cmapi.Certificate{
+				Spec: cmapi.CertificateSpec{
+					EmailSANs:  []string{"alice@example.com"},
+					SecretName: "abc",
+					IssuerRef:  validIssuerRef,
+				},
+			},
+		},
+		"invalid certificate with incorrect email": {
```

Comment:
Is specifying this field with a `mailto` prefix valid? Can you add a test case to ensure it isn't?